readme.md

```diff
@@ -32,17 +33,18 @@ Once you are confident that everything works as expected you can do the official
 7. Unpause.
 8. Deposit the withdrawn amount. 
 9. Harvest again. 
-10. Transfer Ownership of the Vault and Strategy contracts to the Vault and Strat owners for the respective chains. 
+10. Switch to harvest-on-deposit depednding on chain gas fees. 
+11. Transfer Ownership of the Vault and Strategy contracts to the Vault and Strat owners for the respective chains. 
 
 Check for more addresses at https://github.com/beefyfinance/address-book
```

Comment:
This repo is archived, and the address-book package now lives under beefy-api.
```suggestion
Check for more addresses at https://github.com/beefyfinance/beefy-api/tree/master/packages/address-book
```

Reply:
ty @cartman-dev